apps/docs/stories/MetricsCard.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { MetricsCard, MetricsCardItem, MetricsCardProps, MetricType } from '@ror/react/src/components/metrics-card'
+import { ComponentType } from 'react'
+import { Button } from '@ror/react'
+
+const meta = {
+    title: 'ui/MetricsCard',
+    component: MetricsCard as ComponentType<MetricsCardProps>,
+    parameters: {
+        layout: 'centered',
+    },
+    tags: ['autodocs'],
+    argTypes: {
+        item: { control: { type: 'object' }, description: 'Specify the item for the metrics card' },
```

Comment:
Can the description mention what interface the item should adhere to? Right now the description is a bit ambiguous.